pkg/execute/plugin_executor.go

```diff
@@ -82,6 +95,7 @@ func (e *PluginExecutor) Execute(ctx context.Context, bindings []string, slackSt
 		Context: executor.ExecuteInputContext{
 			IsInteractivitySupported: cmdCtx.Platform.IsInteractive(),
 			SlackState:               slackState,
+			KubeConfig:               kubeconfig,
```

Comment:
Should all executor plugins get a kubeconfig?
If we add a new field to `api.Metadata`, can plugin executor access that field?

Reply:
As it is in AC of the task:

> Make sure that rbac property is optional
>    If the rbac context is not provided (as a result, rbac.user|group.type is empty), Botkube won't create a temporary Kubeconfig for a given plugin.

So no, if context.rbac is not provided, we won't create kubeconfig. So this property should be optional (probably a pointer).